src/main/scala/tech/cryptonomic/conseil/Lorre.scala

```diff
@@ -531,7 +532,7 @@ object Lorre extends App with TezosErrors with LazyLogging with LorreAppConfig w
      */
     def process(
         ids: Map[AccountId, BlockReference],
-        votingData: List[(BlockHash, List[Voting.BakerRolls])],
+        votingData: Map[BlockHash, List[Voting.BakerRolls]],
```

Comment:
The idea to use a map here is to avoid a linear search. Now we should have a simple lookup by key, but I can't see it applied in the following code

Reply:
sure, I forgot to use it and changed it accordingly